src/rcl_lifecycle_bindings.cpp

```diff
@@ -223,24 +224,8 @@ NAN_METHOD(GetLifecycleSrvNameAndHandle) {
   Nan::Set(named_srv_obj, Nan::New("name").ToLocalChecked(),
            Nan::New(service_name).ToLocalChecked());
 
-  // Note: lifecycle Services are created and managed by their
-  // rcl_lifecycle_state_machine. Thus we must not manually
-  // free the lifecycle_state_machine's service pointers.
-  // To accomplish this we create srv_handle instances with
-  // the free_ptr parameter of false. Failing to do this results
-  // in a double free() error.
-  auto srv_handle =
-      RclHandle::NewInstance(service, nullptr, [] { return RCL_RET_OK; });
+  auto srv_handle = RclHandle::NewInstance(service, nullptr, nullptr);
   RclHandle* rclHandle = RclHandle::Unwrap<RclHandle>(srv_handle);
```

Comment:
With the new deleter design I think this line is no longer needed.

Reply:
Yes, and I want to get some meaningful comments back here:

  // Note: lifecycle Services are created and managed by their
  // rcl_lifecycle_state_machine. Thus we must not manually
  // free the lifecycle_state_machine's service pointers.